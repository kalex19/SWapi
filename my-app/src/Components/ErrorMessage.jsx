import React, { Component } from 'react';
import './ErrorMessage.scss';

const ErrorMessage = () => {
	return (
		<div className="errorContainer">
			<h1>Select A Category</h1>
			<p>No Favorites Available</p>
		</div>
	);
};

export default ErrorMessage;
