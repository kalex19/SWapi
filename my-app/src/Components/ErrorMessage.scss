@import '../variables.scss';

.errorContainer {
  display: flex;
  justify-content: space-evenly;
  line-height: 30px;
  margin: 3% 0;
  text-shadow: $main-shadow 1px 1px;
}

.errorContainer p {
  font-size: 2em;
}