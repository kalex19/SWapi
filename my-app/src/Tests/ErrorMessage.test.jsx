import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import ErrorMessage from '../src/ErrorMessage';

describe('ErrorMessage', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<ErrorMessage />);
	});

	it('should mock snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
