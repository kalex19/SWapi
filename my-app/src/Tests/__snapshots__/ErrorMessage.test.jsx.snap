// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should mock snapshot 1`] = `ShallowWrapper {}`;
